Make more clear in the example that conditions accumulate (#836)
elastic/observability-docs: docs/en/observability/metrics-threshold-alert.asciidoc

the metric, and by including a warning threshold value, you can be alerted on multiple threshold values based on severity scores.
To help you determine which thresholds are meaningful to you, the preview charts provide a visualization.

> In this example, the conditions state that you will receive a critical alert for hosts with a CPU usage of 120% or above and a warning alert if CPU usage is 100% or above. Note that you will receive a critical alert only if memory usage is 20% or above, as per the second condition.

Reviewer: @jsanz Today I re-read the updated paragraph and realized that the second sentence still needs a clarification as it applies to warning and critical alerts, not critical alert only.
It should read `Note that you will receive an alert only if memory usage is 20% or above, as per the second condition.`

Apologies for a nitpick. I hope it makes a sense.

Author: Created https://github.com/elastic/observability-docs/pull/841 thanks for the suggestion. Nothing to apologize for, nitpicks are great!!

> In this example, the conditions state that you will receive a critical alert for any any hosts that have a CPU usage of 120% or above and a warning alert if CPU usage is 100% or above. Note that you will receive a critical alert only if memory usage is 20% or above, as per the second condition.

Reviewer 2: ```suggestion
In this example, the conditions state that you will receive a critical alert for hosts with a CPU usage of 120% or above and a warning alert if CPU usage is 100% or above. Note that you will receive a critical alert only if memory usage is 20% or above, as per the second condition.
```
